### DISCUSSION TITLE
With the splitting of Neo Store And Droidify, we should have an explaination/comparison of the two

### DISCUSSION THREAD

**Discussion by MysteriousOrb2000** [NeoApplications/Neo-Store#240] [Ideas]
Many people are already confused between the two. I was thinking on the front page of the Neo-Store [README.md,](https://github.com/NeoApplications/Neo-Store#readme) we could have an explanation and a comparison between the two to clear up any confusions.

**Comment by veganomy** [comment]
I'm a Droid-ify user since it's existence. Tried Neo Store and it literally felt like a cluster mess. Duplicate panes, gesture based card like user interface, vertical tabs, all of them feel like too much to handle.  
Droid-ify seems very simple to use, no cluttered UI, exists the only features that are core and needed.

**Comment by Daedalus007** [comment]
Droid-ify convert here.  After bashing my head against the mess of Neo Store I'm a believer now.
- Updates that actually work as expected.
- Background update checks that actually work as expected.
- Updates that don't stutter, freeze, randomly disappear/reappear notifications, none of that garbage.
- Downloads and installs that work the first time, every time, without errors or issues.

About the only downside isn't really Droid-ify's fault:
- There are about 3 repositories that refuse to update: DivestOS, DivestOS Unofficial, and PurpleI2P; DivestOS is a defunct Android custom ROM that no longer exists but for some reason F-Droid hasn't deactivated/removed those repositories despite said repositories no longer existing nor allowing sync/downloads of the older apps they hosted.
